packages/terra-list/src/MultiSelectList.jsx

```diff
@@ -154,20 +189,19 @@ class MultiSelectList extends React.Component {
     return this.props.children.length;
   }
 
-  unusedVariables(variable) {
-    return variable === this;
-  }
-
   render() {
-    const { children, isDivided, onChange, maxSelectionCount, ...customProps } = this.props;
+    const { children, isDivided, ...customProps } = this.props;
```

Comment:
I think it would be good to decide a consistent approach to handling params in this situation. This logic occurs in a few places and are handled a bit differently each time.